### PR TITLE
fixes an exploit with picking up projectiles outside of timestop

### PR DESCRIPTION
**Pull request by Intigracy**
Same check as for attack_hand.

### CODE REVIEW

File: code/modules/projectiles/projectile.dm [hunk 1]
```diff
@@ -676,6 +676,10 @@ var/list/impact_master = list()
 	if(timestopped)
 		..()
 
+/obj/item/projectile/verb_pickup(mob/user)
```
Comment (clusterfack) [inline code/modules/projectiles/projectile.dm]: What

**Comment by Exxion** [comment]
`verb_pickup()` calls `attack_hand()` to do the actual picking up so this is unnecessary
(Unless that was changed since I last checked)

**Comment by Exxion** [comment]
Yes, it does for everything except MoMMIs

**Comment by Intigracy** [comment]
Then it shouldn't even work outside of timestop.

**Pull request closed without merging**